cmd/secret/cmd/root.go

```diff
@@ -252,6 +335,7 @@ func (r *root) serverCmd() *cobra.Command {
 			go func(ctx context.Context) {
 				defer close(shutdownCh)
 				ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+				r.logger.Infof("ctx-shutdown-pdb after connection %v", ctx)
```

Comment:
Probably wrong logger is taken
```suggestion
				logger.Infof("ctx-shutdown-pdb after connection %v", ctx)
```